### DISCUSSION TITLE
Question about Pull Requests

### DISCUSSION THREAD

**Discussion by rh0dy** [wazuh/wazuh#16026] [Q&A] [answered]
What's the approach for getting attention on a PR?  I think https://github.com/wazuh/wazuh/pull/15582 would be quite a nice addition to `4.4` but not sure if it's lacking anything in order for it to get a review.

**Comment by jctello** [comment, marked as answer]
Hi @rh0dy , 
First I would like to thank you for your contribution, it is clear that this will provide value to the Wazuh community.

Wazuh is an Open Source project and we welcome contributions from the community like yours, however most development is done by Wazuh staff which have an internal process for reviewing all changes and prioritizing the work.

We often look for requests from the community (both for development or review) and triage their priority based on the perceived impact that they would have.

I'll let the team internally aware of this PR so it may be merged into a future version.
